CHANGELOG.md

```diff
@@ -4,11 +4,16 @@
  - Added support for validator stacking. This breaks backwards compatability. See https://github.com/lovasoa/marshmallow_dataclass/issues/91.
     ### What this means:
     ```python
+<<<<<<< HEAD
+    CustomType = NewType("CustomType", str, validate=marshmallow.validate.Length(min=3))
+=======
         CustomType = NewType("CustomType", str, validate=marshmallow.validate.Length(min=3))
+>>>>>>> 55797124700e269c817c9ee56785c0b4d76b5a1d
```

Comment:
Looks like you accidentally committed a conflict

Reply:
Yup, VSCode is going crazy. Working on fixing that